dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -154,12 +154,30 @@ def pd_progress
     require 'cdo/properties'
     locals_options = Properties.get("pd_progress_#{script.id}")
     if locals_options
-      render locals: locals_options.symbolize_keys
+      render locals: {pd_script_data: locals_options.symbolize_keys}
     else
       render layout: 'application', text: "PD progress data not found for #{script.name}", status: 404
     end
   end
 
+  def online_pd
+    authorize! :read, :reports
+
+    @scripts = []
+    SeamlessDatabasePool.use_persistent_read_connection do
+      @scripts = Script.where('properties LIKE ?', %q[%"pd":true%]).pluck(:id)
```

Comment:
Style/UnneededPercentQ: Use `%q` only for strings that contain both single quotes and double quotes.